BearlyHumans/squirrelville
# Camera fixes

- Camera behaves like MC 3rd person (zooms in from collisions)
- Move is instant for NoClippingEverLayers
- Move is gradual for ZoomToAvoidLayers
- Other layers are ignored by camera
- Model goes invisible when too close
- FP mode scene

Test:
* Change layers to desired options
* Try to put camera in something = should zoom forward to avoid, speed dependant on layer
* Zoom really close to squirrel = it disappears
* Try FP mode = laugh

Notes/Bugs:
> We were using tags for camera behavior in the past, and are now using layers, so most objects have arbitrary layers that don't give the desired behavior.
